api/v1alpha3/vspherevm_types.go

```diff
@@ -44,6 +44,9 @@ type VSphereVMSpec struct {
 	// this CRD as unstructured data.
 	// +optional
 	BiosUUID string `json:"biosUUID,omitempty"`
+
+	// Insecure is a flag that disables TLS peer verification.
+	Insecure bool `json:"insecure,omitempty"`
```

Comment:
@alexander-demichev - we're looking at deprecating all of the `insecure` knobs. maybe you'd like to tackle this insead ?